### DISCUSSION TITLE
need help to mount rootfs with shared mode

### DISCUSSION THREAD

**Discussion by calvin2021y** [containers/podman#16438] [Q&A]
I am runt a rootless podman with `--rootfs /app/rootfs`, after start 

```sh
findmnt -o PROPAGATION /
PROPAGATION
private
```

is there a way to start podman container with shared rootfs mount ?

**Comment by mheon** [comment]
Not at present, but I also see no reason we couldn't add support. Please file an RFE.

**Comment by rhatdan** [comment]
Adding `--rootfs /app/rootfs:shared` would be easy, or `--rootfs /app/rootfs:slave`

**Comment by giuseppe** [comment]
keep in mind that shared will propagate mounts only to the rootless mount namespace (the environment you get with `podman unshare`)

**Comment by giuseppe** [comment]
one trick you can currently use is to add a shared volume, so podman will automatically use the shared propagation for the container rootfs as well:

```
$ podman run -v /:/root:shared --rm fedora findmnt -o PROPAGATION /
PROPAGATION
shared
```
